rotkehlchen/history/events.py

```diff
@@ -536,3 +538,24 @@ def fail_history_cb(error_msg: str) -> None:
             defi_events,
             ledger_actions,
         )
+
+    def query_kraken_staking_events(
+            self,
+            filter_query: HistoryEventFilterQuery,
+            only_cache: bool,
+    ) -> List[StakingEvent]:
+        """Queries staking events for kraken.
+        If only_cache is given then only trades cached in the DB are returned.
+        No service is queried.
+        """
+        exchanges_list = self.exchange_manager.connected_exchanges.get(Location.KRAKEN)
+        if exchanges_list is None:
+            return []
+
+        if only_cache is not True:
```

Comment:
```suggestion
        if only_cache is False:
```
![giphy](https://user-images.githubusercontent.com/1658405/148200802-eeb7e444-b3a3-4712-b102-6803b64f22ae.gif)

Reply:
https://github.com/rotki/rotki/blame/dc6cb172a726f4af4f114c32aa0f014a84479736/rotkehlchen/history/events.py#L295 :clown_face: